plugins/plugin-editor/src/lib/open.ts

```diff
@@ -19,8 +19,6 @@ import * as Debug from 'debug'
 import * as path from 'path'
 import * as events from 'events'
 
-import { editor } from 'monaco-editor'
```

Comment:
this is a buggy false positive by the lint tool. it incorrectly ignores the use of an imported typescript namespace